tools/serverpod_cli/lib/src/generator/open_api/open_api_definition.dart

```diff
@@ -0,0 +1,200 @@
+library openapi_definition;
+
+import 'package:recase/recase.dart';
+
+import '../../../analyzer.dart';
+import '../../analyzer/dart/definitions.dart';
+import '../../logger/logger.dart';
+
+part 'open_api_objects.dart';
+part 'open_api_utils.dart';
+part 'open_api_extensions.dart';
+
+// ignore_for_file: public_member_api_docs, sort_constructors_first
```

Comment:
It's a bit dangerous to leave these in the code as we want everything documented and follow the lints everywhere. There should be a very code case for ignoring the lint rules.